internal/info/info.go

```diff
@@ -150,25 +230,25 @@ func (d Detail) String() string {
 }
 
 // Get returns parased Detail object.
-func (d Detail) Get() Detail {
-	d.prepare()
+func (i info) Get() Detail {
+	i.prepare()
 	valdRepo := fmt.Sprintf("github.com/%s/%s", Organization, Repository)
-	defaultURL := fmt.Sprintf("https://%s/tree/%s", valdRepo, d.GitCommit)
+	defaultURL := fmt.Sprintf("https://%s/tree/%s", valdRepo, i.detail.GitCommit)
 
-	d.StackTrace = make([]StackTrace, 0, 10)
-	for i := 3; ; i++ {
-		pc, file, line, ok := runtime.Caller(i)
+	i.detail.StackTrace = make([]StackTrace, 0, 10)
+	for j := 3; ; j++ {
+		pc, file, line, ok := i.rtCaller(j)
```

Comment:
please check rt.Caller is nil or not here

Reply:
should we check it on `New()` or here?

Reply:
sounds good, please check it in the construction method.

Reply:
fixed :) thank you